applications/dashboard/addon.json

```diff
@@ -39,7 +39,7 @@
                 "react-redux",
                 "react-router-dom",
                 "redux",
-                "redux-observable",
+                "redux-thunk",
```

Comment:
Does this belong in the rebuild? I would assume not.

Reply:
See the updated description